README.md

```diff
@@ -339,106 +339,348 @@ alpaca.deleteFromWatchlist('xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx', "AAPL").then(
+  config?: any,
+): Promise<AlpacaSnapshot>;
 ```
+```ts
+getSnapshots(
+  symbols: Array<string>,
+  config?: any,
+): Promise<AlpacaSnapshot[]>;
+```
+##### Example
+```ts
+const snapshot = await alpaca.getSnapshot("TSLA");
+console.log(snapshot);
+```
+###### Output:
+```python
+{
+  symbol: 'TSLA',
+  LatestTrade: {
+    Timestamp: '2022-04-19T10:09:23.844940801Z',
+    Exchange: 'Q',
+    Price: 1003,
+    Size: 501,
+    Conditions: [ '@', 'F', 'T' ],
+    ID: 1861,
+    Tape: 'C'
+  },
+  LatestQuote: {
+    Timestamp: '2022-04-19T10:10:09.139921353Z',
+    AskExchange: 'Q',
+    AskPrice: 1004.38,
+    AskSize: 1,
+    BidExchange: 'Q',
+    BidPrice: 1001,
+    BidSize: 3,
+    Conditions: [ 'R' ],
+    Tape: 'C'
+  },
+  MinuteBar: {
+    Timestamp: '2022-04-19T10:09:00Z',
+    OpenPrice: 1003,
+    HighPrice: 1003,
+    LowPrice: 1003,
+    ClosePrice: 1003,
+    Volume: 647,
+    TradeCount: 17,
+    VWAP: 1003.071345
+  },
+  DailyBar: {
+    Timestamp: '2022-04-18T04:00:00Z',
+    OpenPrice: 989.19,
+    HighPrice: 1014.92,
+    LowPrice: 973.41,
+    ClosePrice: 1004.29,
+    Volume: 17209682,
+    TradeCount: 543314,
+    VWAP: 997.42604
+  },
+  PrevDailyBar: {
+    Timestamp: '2022-04-14T04:00:00Z',
+    OpenPrice: 998.51,
+    HighPrice: 1012.7099,
+    LowPrice: 982.19,
+    ClosePrice: 985,
+    Volume: 19449944,
+    TradeCount: 579328,
+    VWAP: 991.712944
+  }
+}
+```
+More detailed examples of stock data endpoints can be found in this [file](https://github.com/alpacahq/alpaca-trade-api-js/blob/master/examples/marketdataV2.ts). 
+
+##### News
+```ts
+ getNews(
+   options: GetNewsParams,
+   config?: any,
+ ): Promise<AlpacaNews[]>;
+```
+Details of the `options` parameter can be found [here](https://github.com/alpacahq/alpaca-trade-api-js/blob/687e58c8ff03cc70b56fc57844d78ca7801e0f85/lib/resources/datav2/rest_v2.ts#L714).
+##### Example
+```ts
+ const news = await alpaca.getNews({});
+ console.log(news[0]);
+```
+###### Output:
+```python
+{
+  ID: 26682466,
+  Headline: 'Plug Power Enters Agreement With Walmart Pursuant To Which The Co. Will Deliver Liquid Green Hydrogen To New And Existing Walmart Sites In The United States',
+  Author: 'Bill Haddad',
+  CreatedAt: '2022-04-19T10:09:38Z',
+  UpdatedAt: '2022-04-19T10:09:39Z',
+  Summary: '',
+  URL: 'https://www.benzinga.com/news/22/04/26682466/plug-power-enters-agreement-with-walmart-pursuant-to-which-the-co-will-deliver-liquid-green-hydrogen',
+  Images: [],
+  Symbols: [ 'PLUG', 'WMT' ],
+  Source: 'benzinga'
+}
+```
+### Data API - Crypto
+All the functions are similar to the stock ones. 
+#### Trades
+* `getCryptoTrades`
+* `getLatestCryptoTrade`
+* `getLatestCryptoTrades`
+
+#### Quotes
+* `getCryptoQuotes`
+* `getLatestCryptoQuote`
+* `getLatestCryptoQuotes`
+
+#### Bars
+* `getCryptoBars`
+* `getLatestCryptoBar`
+* `getLatestCryptoBars`
+
+#### XBBOs
+* `getLatestCryptoXBBO`
+* `getLatestCryptoXBBOs`
+
+#### Snapshots
+* `getCryptoSnapshots`
+
```

Comment:
Can we please still have type signatures here? Easier for people who just land on this page only for crypto